src/System.Security.Cryptography.X509Certificates/tests/CertificateCreation/CertificateRequestChainTests.cs

```diff
@@ -398,12 +398,9 @@ private static X509Certificate2 CloneWithPrivateKey(X509Certificate2 cert, Asymm
                 intermed2Serial[1] = 2;
                 leafSerial[1] = 1;
 
-                using (RandomNumberGenerator rng = RandomNumberGenerator.Create())
-                {
-                    rng.GetBytes(intermed1Serial, 2, intermed1Serial.Length - 2);
-                    rng.GetBytes(intermed2Serial, 2, intermed2Serial.Length - 2);
-                    rng.GetBytes(leafSerial, 2, leafSerial.Length - 2);
-                }
+                RandomNumberGenerator.Fill(intermed1Serial.AsSpan(2, intermed1Serial.Length - 2));
+                RandomNumberGenerator.Fill(intermed2Serial.AsSpan(2, intermed2Serial.Length - 2));
+                RandomNumberGenerator.Fill(leafSerial.AsSpan(2, leafSerial.Length - 2));
```

Comment:
FYI: These could just be, e.g. `leafSerial.AsSpan(2)`, since "until the current last item" is implied.

Not worth re-spinning CI, though.